lib/control-connection.js

```diff
@@ -95,10 +95,12 @@ ControlConnection.prototype.init = function (callback) {
           var portSeparatorIndex = name.lastIndexOf(']:');
           return addHost(name.substr(1, portSeparatorIndex - 1), name.substr(portSeparatorIndex + 2), eachNext);
         }
-        if (name.indexOf('.') > 0 && name.indexOf(':') > 0) {
+        if (name.indexOf(':') > 0) {
```

Comment:
nice catch @tolbertam ! I'll move this to a repo branch and we can apply your patch there.